spec/initializers/active_support/cache/redis_store_spec.rb

```diff
@@ -0,0 +1,51 @@
+require "rails_helper"
+
+RSpec.describe ActiveSupport::Cache::RedisStore do
+  describe "ActiveSupport::Cache::RedisCacheStore" do
+    let(:redis_url) { "redis://localhost:6379" }
+    let(:redis_client) { Redis.new(url: redis_url) }
```

Comment:
This should read the url from the already configured one. Actually you don't need the URL at all, you can just use `Redis.current` but even better, you don't need any of it. You can just use: "let(:cache) { ActiveSupport::Cache.lookup_store(:redis_cache_store) }"

See:

```ruby
[9] pry(main)> ActiveSupport::Cache.lookup_store(:redis_cache_store).redis
=> #<Redis client v4.1.3 for redis://127.0.0.1:6379/0>
```